gui/wxpython/datacatalog/tree.py

```diff
@@ -1060,6 +1145,18 @@ def _popupMenuMapset(self):
         self.PopupMenu(menu)
         menu.Destroy()
 
+    def _popupMenuLocation(self):
+        """Create popup menu for locations"""
+        menu = Menu()
+        genv = gisenv()
+        currentLocation, currentMapset = self._isCurrent(genv)
```

Comment:
not used, it is actually causing error when right clicking on current location, just remove it